shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -0,0 +1,524 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
`s/2013/2018/`, @Hixie can confirm the exact circumstances where we preserve old dates, but IIRC new code that isn't a move or move+tweak of existing code gets a new date.

Reply:
The CI is there just to make sure a human verifies any changes to the licences we collect, not to prevent the addition of licences. It'll emit a diff which you can apply via `patch`.

Reply:
@cbracken it wasn't the license file itself that was broken.  The CI literally has a condition that checks the "license count" at the end of the file and specifically warns against adding more licenses.

Reply:
Unless it's important to do otherwise, I'd like to commit without this change because I think this will likely involve a bit of back and forth and it's easy to change these headers later.

Reply:
> The CI literally has a condition that checks the "license count" at the end of the file and specifically warns against adding more licenses.

Correct that happens here: https://github.com/flutter/engine/blob/master/ci/licenses.sh